modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/NavigationsUtils.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+export const moveRight = (container, navigationController) => {
+	const getPosition = container.attr('transform');
+	const scale = getPosition.match(/(-?[0-9]+[.,-\s]*)+/g);
```

Comment:
This regex should be moved outside of the function as a constant and then also provide a comment above the regex explaining what it is matching for, otherwise it is really hard to know what this regex is for.

Reply:
Same comment applied for the other functions below

Reply:
thank you